File: AICodingWeb/Models/Person.cs
using System;
using System.Collections.Generic;

namespace AICodingWeb.Models
{
    public class Person
    {
        public int PersonID { get; set; }
        public string FullName { get; set; }
        public string PreferredName { get; set; }
        public string SearchName { get; set; }
        public bool IsPermittedToLogon { get; set; }
        public string LogonName { get; set; }
        public bool IsExternalLogonProvider { get; set; }
        public byte[] HashedPassword { get; set; }
        public bool IsSystemUser { get; set; }
        public bool IsEmployee { get; set; }
        public bool IsSalesperson { get; set; }
        public string UserPreferences { get; set; }
        public string PhoneNumber { get; set; }
        public string FaxNumber { get; set; }
        public string EmailAddress { get; set; }
        public byte[] Photo { get; set; }
        public string CustomFields { get; set; }
        public string OtherLanguages { get; set; }
        public int LastEditedBy { get; set; }
        public DateTime LastEditedWhen { get; set; }

        public virtual ICollection<Customer> PrimaryContactCustomers { get; set; }
        public virtual ICollection<Customer> AlternateContactCustomers { get; set; }
        public virtual ICollection<Supplier> PrimaryContactSuppliers { get; set; }
        public virtual ICollection<Supplier> AlternateContactSuppliers { get; set; }
        public virtual ICollection<Order> PickedByOrders { get; set; }
        public virtual ICollection<Order> SalespersonOrders { get; set; }
        public virtual ICollection<Invoice> ContactPersonInvoices { get; set; }
        public virtual ICollection<Invoice> AccountsPersonInvoices { get; set; }
        public virtual ICollection<Invoice> SalespersonInvoices { get; set; }
        public virtual ICollection<Invoice> PackedByPersonInvoices { get; set; }
        public virtual ICollection<PurchaseOrder> ContactPersonPurchaseOrders { get; set; }
    }
} 

File: AICodingWeb/aicoding_web/app/models/person.py
from datetime import datetime
from app import db
from app.models.base import BaseModel

class Person(db.Model, BaseModel):
    """Person model"""
    
    __tablename__ = 'People'
    __table_args__ = {'schema': 'Application'}
    
    PersonID = db.Column(db.Integer, primary_key=True)
    FullName = db.Column(db.String(50), nullable=False)
    PreferredName = db.Column(db.String(50), nullable=False)
    SearchName = db.Column(db.String(101), nullable=False)
    IsPermittedToLogon = db.Column(db.Boolean, nullable=False)
    LogonName = db.Column(db.String(50))
    IsExternalLogonProvider = db.Column(db.Boolean, nullable=False)
    HashedPassword = db.Column(db.String(50))
    IsSystemUser = db.Column(db.Boolean, nullable=False)
    IsEmployee = db.Column(db.Boolean, nullable=False)
    IsSalesperson = db.Column(db.Boolean, nullable=False)
    UserPreferences = db.Column(db.JSON)
    PhoneNumber = db.Column(db.String(20))
    FaxNumber = db.Column(db.String(20))
    EmailAddress = db.Column(db.String(256))
    Photo = db.Column(db.LargeBinary)
    CustomFields = db.Column(db.JSON)
    OtherLanguages = db.Column(db.JSON)
    LastEditedBy = db.Column(db.Integer, db.ForeignKey('Application.People.PersonID'), nullable=False)
    ValidFrom = db.Column(db.DateTime, nullable=False)
    ValidTo = db.Column(db.DateTime, nullable=False)
    
    def __repr__(self):
        return f'<Person {self.PersonID}>' 